test/test_storage.cpp

```diff
@@ -928,11 +928,12 @@ void test_fastresume(bool const test_deprecated)
 		p.storage_mode = storage_mode_sparse;
 		error_code ignore;
 		torrent_handle h = ses.add_torrent(std::move(p), ignore);
-		TEST_CHECK(exists(combine_path(p.save_path, "temporary")));
```

Comment:
It is noteworthy that this "use-after-move" bug also went unnoticed, as `p` was not actually moved.